nameserver/status.go

```diff
@@ -32,12 +33,15 @@ func NewStatus(ns *Nameserver, dnsServer *DNSServer) *Status {
 			entry.ContainerID,
 			entry.Addr.String(),
 			entry.Version,
-			entry.Tombstone})
+			entry.Tombstone,
+		})
 	}
 
 	return &Status{
 		dnsServer.domain,
+		dnsServer.upstream.Servers,
 		dnsServer.address,
 		dnsServer.ttl,
-		entryStatusSlice}
+		entryStatusSlice,
+	}
```

Comment:
Sorry to be a pain, but can you undo it? The entire rest of the status code is written in this style, and *if* we're going to change it we should do so uniformly in a separate PR...